fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java

```diff
@@ -83,7 +75,9 @@ public EntityFieldConfigurationApiResources(final PlatformSecurityContext contex
 	@GET
 	@Consumes({ MediaType.APPLICATION_JSON })
 	@Produces({ MediaType.APPLICATION_JSON })
-	public String getAddresses(@PathParam("entity") final String entityname, @Context final UriInfo uriInfo) {
+	@ApiOperation(value = "Lists Entity Field Configuration", httpMethod = "GET", notes = "It Lists Entity Field Configuration")
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", response = EntityFieldConfigurationApiResourcesSwagger.GetEntityFieldConfigurationResponse.class, responseContainer = "List")})
```

Comment:
1. Indentation is not correct!
2. Refractor the name of Inner Class to GetFieldConfigurationEntityResponse